include/alpaka/vec/Vec.hpp

```diff
@@ -511,8 +512,7 @@ namespace alpaka
         }
 
     private:
-        // 16 Byte alignment for usage inside of CUDA kernels.
-        alignas(16u) TSize m_data[TDim::value];
+        alignas(ALPAKA_OPTIMAL_ALIGNMENT_SIZE(TSize[TDim::value])) TSize m_data[TDim::value];
```

Comment:
The problem is that the members in `WorkDivMembers` are not aligned:
align members in [WorkDivMembers](https://github.com/ComputationalRadiationPhysics/alpaka/blob/develop/include/alpaka/workdiv/WorkDivMembers.hpp#L127-L129)  and use **Version 1**:
```C++
alignas(ALPAKA_OPTIMAL_ALIGNMENT(Vec<TDim, TSize>)) Vec<TDim, TSize> m_gridBlockExtent;
alignas(ALPAKA_OPTIMAL_ALIGNMENT(Vec<TDim, TSize>)) Vec<TDim, TSize> m_blockThreadExtent;
alignas(ALPAKA_OPTIMAL_ALIGNMENT(Vec<TDim, TSize>)) Vec<TDim, TSize> m_threadElemExtent;
```
results in vectorized code:

```C++
///bigdata/hplsim/scratch/widera/alpaka/include/alpaka/workdiv/WorkDivMembers.hpp:77             {}
	.loc 6 77 1
	ld.u32 	%r1, [%rd2+8];
	ld.u32 	%r2, [%rd2+4];
	ld.u32 	%r3, [%rd2];
	st.v2.u32 	[%rd1], {%r3, %r2};
	st.u32 	[%rd1+8], %r1;
	ld.u32 	%r4, [%rd2+16];
	ld.u32 	%r5, [%rd2+20];
	ld.u32 	%r6, [%rd2+24];
	st.u32 	[%rd1+24], %r6;
	st.u32 	[%rd1+20], %r5;
	st.u32 	[%rd1+16], %r4;
	ld.u32 	%r7, [%rd2+32];
	ld.u32 	%r8, [%rd2+36];
	ld.u32 	%r9, [%rd2+40];
	st.u32 	[%rd1+40], %r9;
	st.u32 	[%rd1+36], %r8;
	st.u32 	[%rd1+32], %r7;
```

**Members in all structs must aligned to support auto !**